Project.toml

```
@@ -18,6 +18,15 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 julia = "1"
+Calculus = "0.3.1"
```

Does it not work with the current Calculus version? Maybe you want
```suggestion
Calculus = "0.3.1, 0.4"
```

---

I wrote 3 by mistake, meant 4 :)